js/whitebit.js

```diff
@@ -463,22 +464,23 @@ module.exports = class whitebit extends Exchange {
         //      }
         //
         const currenciesIds = Object.keys (response);
-        const withdrawFees = {};
-        const depositFees = {};
+        const result = {};
         for (let i = 0; i < currenciesIds.length; i++) {
```

Comment:
Since whitebit returns a coin-indexed object, I think we can do the opposite here, iterate over the user-provided list and access the info directly,

rough example:

```Javascript

for (let i = 0; i < userProvidedCodes.length; i++) {
      const currency = userProvideCodes[i];
      const data = response[currency];
       (...)
}
```
wdyt?